src/components/greeting-container/greeting-container.less

```diff
@@ -0,0 +1,19 @@
+.info-container {
```

Comment:
We want to start using styled components for things like this, but I don't think it's installed on this repo so we can do it later, after a merge from frontend-scaffold